scripts/download-akzidenz.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-console */
+const path = require('path');
+const https = require('https');
+const fs = require('fs');
+
+const download = (url, destDir) => {
+  const destFileName = path.join(destDir, path.basename(url));
+  const destFile = fs.createWriteStream(destFileName);
+  console.log('Downloading', url, 'to', path.relative(process.cwd(), destFileName));
+
+  return new Promise((resolve, reject) => {
+    https.get(url, (response) => {
+      response.pipe(destFile)
+        .once('error', reject)
+        .on('finish', resolve);
```

Comment:
```suggestion
      resolve(util.promisify(stream.pipeline))(response, destFile);
```

comes with better error handling (`stream.pipeline()` cleans up all streams if one of them errors out, and handles errors from all of them) :slightly_smiling_face: